Sources/MapboxSearch/PublicAPI/FavoriteRecord.swift

```diff
@@ -11,6 +11,9 @@ public struct FavoriteRecord: IndexableRecord, SearchResult, Codable, Equatable
     /// A unique identifier for the geographic feature
     public var mapboxId: String?
 
+    /// Type of the favorite record.
+    public var favoriteType: String?
```

Comment:
What are the semantics of `favoriteType`? Is this entirely user-defined and freely associated? Or should this be an enum with fixed cases?